src/app/directives/resource-panel.tpl.html

```diff
@@ -3,9 +3,21 @@
     <strong>Try-it</strong> is disabled because <strong>baseUri</strong> is not present
   </div>
   <div class="raml-console-resource-panel-wrapper">
-    <documentation class="raml-console-resource-panel-primary" method-info="methodInfo" security-schemes="securitySchemes"></documentation>
+    <documentation
+      class="raml-console-resource-panel-primary"
+      method-info="methodInfo"
+      resource="resource"
+      security-schemes="securitySchemes"></documentation>
 
-    <sidebar ng-show="baseUri"></sidebar>
+    <sidebar
+      ng-show="baseUri"
+      base-uri-parameters="baseUriParameters"
+      collapse-sidebar="collapseSidebar"
+      context="context"
+      method-info="methodInfo"
+      protocols="protocols"
+      resource="resource"
+      security-schemes="securitySchemes"></sidebar>
```

Comment:
Can we put this close tag on a new line with the same indentation as the start of the tag?
It will make it look more like a code block